blacksmith/src/lib.rs

```diff
@@ -123,6 +125,63 @@ impl Blacksmith {
             }
         }
 
+        let mut stable_version = SemVersion::from_str(&blacksmith.stable_version.clone().unwrap());
+
+        // Go backwards in stable versions
+        while let Some(mut previous_stable_version) = stable_version.previous() {
```

Comment:
I'm fine with this approach but it may be simpler to just use https://static.rust-lang.org/manifests.txt. That is up to date and filtering it for stable releases is probably simpler than semver-scanning back and forward.

Reply:
Oh, I didn't know about this file. It would have made my work on this PR way simpler. :)
Yes, I will switch the implementation to use that.

Reply:
Switched to using https://static.rust-lang.org/manifests.txt.